channeldb/graph.go

```diff
@@ -30,7 +30,7 @@ const (
 	// This presents a tradeoff between the number concurrent operations
 	// that can to be applied by a single database transaction and the
 	// overall latency of the changes being applied.
-	GraphBatchCommitInterval = time.Second
+	GraphBatchCommitInterval = 10 * time.Millisecond
```

Comment:
This commit (the one that uses a 10 ms commit interval) should be scrapped right? 

(based on johan's testing thus far)

Reply:
yes, this is just used so that we can test that the batched versions pass all of our unit tests that are currently written with the non-batched versions. the main question we have to decide is whether or not to keep the unbatched versions at all, though it could slow down the unit tests somewhat and possibly lead to flakes since the current timing assumptions may be invalidated due to the additional latency.

another option is to pass in this lower commit latency when the channeldb is instantiated, so that we can use a lower value in testing. @halseth also suggested elsewhere we make this configurable, tho i'm still not sure it needs to be exposed as a CLI/conf option

Reply:
Can we expose it as a conf option for dev or expermental builds only? And default it to 500 ms or whatever